codebase-pipelines/variables.tf

```diff
@@ -37,3 +37,8 @@ variable "services" {
 variable "env_config" {
   type = any
 }
+
+variable "image_build" {
+  type = bool
+  default = true
+}
```

Comment:
In my ticket this was going to be "build_image" which probably reads a little better in the context of a boolean. 

Actually perhaps "requires_image_build" may be better?